docs/guide/ecosystem/server-side-rendering.md

```diff
@@ -119,6 +119,15 @@ If you are searching for a well tested solution, with a large community and nice
 
 Learn more about it on the dedicated [Nuxt](./nuxt.md) page.
 
+## Servue
+[Servue]https://futureaus.github.io/servue/ shares a similar scope to Nuxtjs, but with more freedom and targetted more towards Multi-page-applications using existing server/routing systems such as koa/koa-router and express.js
```

Comment:
Link is not formatted properly.
Stating it is similar to Nuxt is not optimal. It's better to explain what is the difference between and why would one chose over the other. As Nuxt is a framework of its own, I would not compare them at all.